RecoTracker/LSTCore/standalone/README.md

```diff
@@ -1,53 +1,134 @@
-# TrackLooper
+# How to set up standalone LST
 
-
-## Quick Start
-
-
-### Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
+## Setting up LSTPerformanceWeb (only for lnx7188 and lnx4555)
 
 For lnx7188 and lnx4555 this needs to be done once
 
     cd /cdat/tem/${USER}/
     git clone git@github.com:SegmentLinking/LSTPerformanceWeb.git
 
-### Setting up container (only for lnx7188)
+## Setting up container (only for lnx7188)
 
 For lnx7188 this needs to be done before compiling or running the code:
 
     singularity shell --nv --bind /mnt/data1:/data --bind /data2/segmentlinking/ --bind /opt --bind /nfs --bind /mnt --bind /usr/local/cuda/bin/ --bind /cvmfs  /cvmfs/unpacked.cern.ch/registry.hub.docker.com/cmssw/el8:x86_64
 
-### Setting up the code
+## Setting up LST
+
+There are two way to set up LST as a standalone, either by setting up a full CMSSW area, which provides a unified setup for standalone and CMSSW tests, or by `sparse-checkout` only the relevant package and using them independent of CMSSW. A CVMFS-less setup is also provided for the second option.
+
+### Setting up LST within CMSSW (preferred option)
+
+```bash
+CMSSW_VERSION=CMSSW_14_2_0_pre4 # Change with latest/preferred CMSSW version
+LST_BRANCH=CMSSW_14_1_0_pre3_LST_X_LSTCore_realfiles_batch1_devel # Change to the appropriate branch
```

Comment:
Yeah, that would be good. Maybe it can just be `master_LST`, we submit PRs to CMSSW from there, and every time they get merged we sync it with the upstream master.

Reply:
I think mkFit has an established way of doing that that works nicely. @slava77 could probably give us the details.

Reply:
mkFit is using `master` in trackreco repository directly for the internal PRs. It's resynced whenever necessary. Also, in mkFit we never merge PRs directly: the same topic branch is submitted to the upstream and once it's reviewed and merged, it will appear as merged in trackreco as well (except that not all developers do it and we end up with PRs that look unmerged but actually are).
there is no CI in mkFit and using `master` is more straightforward.
So, I'm not sure that in our case the is a good reference in mkFit practices.

With CI things get complicated and we need to discuss/decide what's a better mode.

Smth like `CMSSW_14_2_0_pre4_LST_X` that's explicitly coupled to a pre-release is more clear for developers and the CI. The downside is this requires explicit updates to a new branch and pre-release. 

Using `master` for a target can be OK, but the CI would need to be updated to figure out the latest working release and do a rebase of the topic branch for a test, which is problematic. 

Another question is do we want to merge in SegmentLinking/cmssw/master-or-whatever before the full review in the upstream?

Reply:
I was under the assumption that we would submit to CMSSW in batches so that things can move quicker. Either way, I can figure out a way to get the CI to pick the right release if needed.

Reply:
> I was under the assumption that we would submit to CMSSW in batches so that things can move quicker.

It's practical for our todos from #75 
After that part settles it may be more practical to avoid batching.